cmd/juju/storage/storage.go

```diff
@@ -90,14 +89,21 @@ func formatStorageInfo(all []params.StorageInstance) (map[string]StorageInfo, er
 			panic(err) // impossible
 		}
 		// TODO format size nicely (fraction with MGTPEZY suffix, depending on size).
-		output[storageTag.Id()] = StorageInfo{
+		si := StorageInfo{
 			StorageName:   storageName,
 			Location:      one.Location,
-			Owner:         ownerTag.Id(),
 			AvailableSize: one.AvailableSize,
 			TotalSize:     one.TotalSize,
 			Tags:          one.Tags,
 		}
+		owner := ownerTag.Id()
+		ownerColl := output[owner]
```

Comment:
this can be better done as:
if ownerCol1, ok := output[owner];  !ok {
    ownerCol1 = map[string]StroageInfo{}
   output[owner] = ownerCol1
}